### DISCUSSION TITLE
I am trying to config khoj to add my open api key and use chat feature but facing errors

### DISCUSSION THREAD

**Discussion by Uchiha1391** [khoj-ai/khoj#164] [Q&A]
I am able to run khoj --no-gui  but not other commands. Here are the screenshots of the error
Also I run those command in power shell which is running on administrative mode
![image](https://user-images.githubusercontent.com/28823508/219941666-4e2024d7-136f-476c-a598-db5ca864e31f.png)
![image](https://user-images.githubusercontent.com/28823508/219941681-f0fa8beb-a2ed-4ffa-8af4-a9e0e5671269.png)

**Comment by jasmeetchadha** [comment]
Same issue. Cannot run khoj with gui.

**Reply by debanjum** [reply]
To help figure out what is not working:

1. Can you share which version of khoj you're running by runnig `khoj -V`

2. When you say you're able to run `khoj --no-gui`, do you mean you are able to search through your notes as well? Can you share screenshot of what you see when you run `khoj --no-gui -vv`?

**Reply by Karan-GameDev** [reply]
I am not at my pc so can't tell the khoj version but I installed python 3.9 today and khoj --no-gui was working meaning i am able to search with it on localhost:8000.
I will add the missing information tomorrow

**Comment by jasmeetchadha** [comment]
1) 0.3.0a1676245372
2) This is working fine (no-gui)... its the GUI that doesnt work. And I am unable to give it my openai API key (in cmd) to test chat functions

![image](https://user-images.githubusercontent.com/49923268/219971756-dba7cd6b-f6fd-471c-88c9-df63dbf5619d.png)

**Reply by Karan-GameDev** [reply]
Yup, gui doesn't work. I wanted to use chat feature and for that I need to add open ai key. First I thought I need to add that to config file but it's keep saying you don't have permission even though my powershell is in admin mode. Then I tried khoj command to see if there is any gui where I can add my key but it isn't running either.

**Reply by debanjum** [reply]
I see. Was khoj GUI ever working for either of you? Let me investigate why the khoj gui could have broken. 

Until then to test the chat functionality, you can do the following:
1. Directly add your openai-api key to the khoj config file at `C:\Users\<your-windows-username>\.khoj\khoj.yml`. See the second point in [this section](https://github.com/debanjum/khoj#set-your-openai-api-key-in-khoj) to do that
2. Restart khoj server with `khoj --no-gui -vv`
3. Open http://localhost:8000/chat?t=summarize in your browser

**Reply by debanjum** [reply]
> First I thought I need to add that to config file but it's keep saying you don't have permission even though my powershell is in admin mode. 

Not sure why the permissions issue is coming up. Is it only coming up when you start khoj in GUI mode but not in non-gui mode? Which config file did you use? Was it the `.khoj/khoj.yml` file in your home folder? If not, did you start khoj with `khoj --no-gui -vv -c \path\to\your\khoj.yml` See the [sample khoj.yml](https://github.com/debanjum/khoj/blob/master/config/khoj_sample.yml) for reference on how your khoj.yml file should be structured.
